src/utils/Logger.ts

```diff
@@ -69,7 +70,7 @@ class Logger {
     // prevent genuine network errors from being sent to Sentry
     if (!isNetworkError || (this.isNetworkConnected && isNetworkError)) {
       const captureContext = {
-        level: Sentry.Severity.Error,
+        level: 'error' as SeverityLevel,
```

Comment:
Why?

Reply:
This was needed due to the sentry upgrade - [changelog](https://github.com/getsentry/sentry-javascript/blob/master/MIGRATION.md#severity-severitylevel-and-severitylevels).